vim25/types/unreleased.go

```diff
@@ -145,15 +145,29 @@ func init() {
 
 type ClusterClusterRelocatePlacementAction struct {
 	ClusterAction
-	TargetHost   *ManagedObjectReference     `xml:"targetHost,omitempty"`
-	Pool         ManagedObjectReference      `xml:"pool"`
-	RelocateSpec *VirtualMachineRelocateSpec `xml:"relocateSpec,omitempty"`
+	TargetHost        *ManagedObjectReference     `xml:"targetHost,omitempty"`
+	Pool              ManagedObjectReference      `xml:"pool"`
+	RelocateSpec      *VirtualMachineRelocateSpec `xml:"relocateSpec,omitempty"`
+	AvailableNetworks []ManagedObjectReference    `xml:"availableNetworks,omitempty"`
 }
 
 func init() {
 	t["ClusterClusterRelocatePlacementAction"] = reflect.TypeOf((*ClusterClusterRelocatePlacementAction)(nil)).Elem()
 }
 
+type ClusterXClusterInitialPlacementAction struct {
```

Comment:
As we discussed offline, a previously released change has already introduced `ClusterClusterInitialPlacementAction` in "types.go". It seems that in order to avoid the name conflict and keep the new property in the unreleased file, you have to track this new interface `ClusterXClusterInitialPlacementAction` separately.

 If that's the case, I would recommend to keep the same naming pattern for both actions introduced in the vSphere Zone project:
- ClusterXClusterRelocatePlacementAction
- ClusterXClusterInitialPlacementAction

Reply:
Resolved by offline discussion.